src/gui/rss/automatedrssdownloader.ui

```diff
@@ -107,6 +107,17 @@
            </widget>
           </item>
           <item>
+           <widget class="QCheckBox" name="checkSmart">
+            <property name="toolTip">
+             <string>Smart Episode Filter will check the episode number to prevent downloading of duplicates.
+Supports the formats: S01E01, 1x1, 2017.01.01 and 01.01.2017 (Date formats also support - as a separator)</string>
```

Comment:
Just use the following:
```
<string>Smart Episode Filter will check the episode number to prevent downloading of duplicates.
Supports the formats: S01E01, 1x1 and 2017.01.01</string>
```
Seems qtcreator converts '\n' to a real newline and it is alright.